server/datastore/mysql/software_title_display_names.go

```diff
@@ -0,0 +1,74 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/jmoiron/sqlx"
+)
+
+func updateSoftwareTitleDisplayName(ctx context.Context, tx sqlx.ExtContext, teamID *uint, titleID uint, displayName string) error {
+	_, err := tx.ExecContext(ctx, `
+		INSERT IGNORE INTO software_title_display_names
```

Comment:
Any reason for the `IGNORE`? Typically we used it when a `ON DUPLICATE KEY UPDATE` was not used, so it wouldn't fail if it already existed, but since you have an `ON DUPLICATE KEY UPDATE` we should probably not ignore any other consistency error?

Reply:
@mna good catch! I can remove.